core/contextmenu_items.ts

```diff
@@ -554,6 +557,108 @@ export function registerHelp() {
   ContextMenuRegistry.registry.register(helpOption);
 }
 
+/** Registers an option for deleting a workspace comment. */
+export function registerCommentDelete() {
+  const deleteOption: RegistryItem = {
+    // TODO: Before merging, I had to make this a function because I don't think
+    //   the messages were loaded in time. Is that a larger bug?
```

Comment:
Resolve or remove?

Reply:
@maribethb @cpcallen does this look like an issue to you?

Reply:
Discussed over chat. We're going to say this is WAI and if you want your messages to be translated you need to pass them through a function.